collisions_functions.py

```diff
@@ -0,0 +1,197 @@
+"""This module defines atom class and functions used in an elastic
+collision physics simulation.
+"""
+
+import math
+import random
+import numpy
+from random import randrange, uniform
+
+
+# Parameters
+red = (255,0,0)

```

Comment:
If this is constant, there is a strong convention in python of naming it in `ALL_CAPS`

Reply:
Ah yep, I wasn't sure if non pure number structures like tuples still count as constants but makes sense that they can be.